pkg/model/iam/iam_builder.go

```diff
@@ -244,13 +244,12 @@ func (b *PolicyBuilder) BuildAWSPolicyBastion() (*Policy, error) {
 		Version: PolicyDefaultVersion,
 	}
 
-	// Bastion hosts currently don't require any specific permissions.
-	// A trivial permission is granted, because empty policies are not allowed.
-	p.Statement = append(p.Statement, &Statement{
-		Effect:   StatementEffectAllow,
-		Action:   stringorslice.Slice([]string{"ec2:DescribeRegions"}),
-		Resource: resource,
-	})
+	addNodeEC2Policies(p, resource)
+
+	var err error
+	if p, err = b.AddS3Permissions(p); err != nil {
```

Comment:
Should Bastion get all of these permissions in s3? Letting it access config store makes sense for what is happening here, but the secrets seems like something one should avoid.

Reply:
I was wondering the same. Nodeup is still kind of magic to me, so i'm still researching how it works exactly.
I will try to pin point exactly what permission it needs.

Reply:
@olemarkus Removed S3 permission for pki sub-directory